### PR TITLE
float.sh must print exactly three digits after the dot

### PR DESCRIPTION
**Pull request by LaithAlebrahim**
I had some issues with previous with fork installation and etc..  so I reforked and rewrite the code and added new tests and modify other tests to ensure from my code I run `make test` and `make lint` everything works fine for me.
Sorry for interruption. please review it and let me know if you have any concerns.Thanks

@yegor256

### CODE REVIEW

File: tests/metrics/test-multimetric.sh
Comment (LaithAlebrahim) [inline tests/metrics/test-multimetric.sh:1]: I just make `git pull` and found out that you've changed metrices name so I renamed them and it passed the tests

File: tests/steps/test-measure-file.sh
Comment (LaithAlebrahim) [inline tests/steps/test-measure-file.sh:1]: Same as previous comment:
I made `git pull` and found out that you've changed metrices name so I renamed them and it passed the tests

**Comment by yegor256** [comment]
@LaithAlebrahim looks like shellcheck is not happy about your code. Try again locally: `make lint`

**Comment by LaithAlebrahim** [comment]
> @LaithAlebrahim looks like shellcheck is not happy about your code. Try again locally: `make lint`

@yegor256 Should be fixed now ,Hopefully :(

**Comment by yegor256** [comment]
@LaithAlebrahim thanks!

**Pull request merged**